update-imported-docs/README-hi.md

```diff
@@ -0,0 +1,63 @@
+
+# आयातित डॉक्स अपडेट करें
+
+यह स्क्रिप्ट डॉक्स फ़ाइलों को अपडेट करती है जो अन्य रिपॉजिट से उत्पन्न होती हैं।
```

Comment:
`रिपॉजिट` should be replace with `रेपोस` or english for  that is `repos`

Reply:
Other reviewers reviewers needed

Reply:
lets go with the english repos!!!

Reply:
I am sorry, I just checked on the preferred word usage that @mittalyashu has put together it reads: रीपॉज़टॉरी

Reply:
ohh! but here `रिपॉजिट` doesn't feel good if we read the sentence. but it's correct translation for the word `repository`

Reply:
रेपोस or repos or रीपॉज़टॉरी !!